demo/video/sample.env

```diff
@@ -1 +1,2 @@
 VITE_ROOM_TOKEN=<Token from the `api/video/room_tokens` endpoint>
+VITE_ROOM_TOKEN_FULL_PERMISSIONS=<Same kind of token as above, but with all permissions, to demonstrate all features>
```

Comment:
could we keep just one token instead of two?

To ensure that `VITE_ROOM_TOKEN` has all the necessary permissions, we could provide a bash or node script that, given a project id and api token, spits out a token (with all the necessary permissions baked in)